.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+on:
+  pull_request:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: npm ci
+      - name: Generate Bundle
+        run: npm run build:bundle
+        # We need to make sure the checked-in `index.js` actually matches what we expect it to be.
```

Comment:
please put this comment inside the next list item, as it is a comment on the next list item.
not on the list item it where it is currently placed in